shared/styles/style-guide.desktop.js

```diff
@@ -44,7 +44,8 @@ export const globalColors = {
 }
 
 export const globalResizing = {
-  login: {width: 700, height: 580},
+  // Add 25px to login for the OS menubar
+  login: {width: 700, height: 605},
```

Comment:
Signup and login are both 580px in Zeplin, but if I leave it at 580 in globalResizing then I only get ~560px of content, on both OSX and Linux.